harness/determined/load/_native.py

```diff
@@ -0,0 +1,151 @@
+import contextlib
```

Comment:
Why do we still have this file?

Reply:
Because we are not removing `det.experimental.create()` yet, due to customers who are actively using it.

Reply:
added some tests for the `det.experimental.create()` codepath; previously there were none at all.